packs/travisci/actions/setup_hook.yaml

```diff
@@ -0,0 +1,14 @@
+name: setup_hook
```

Comment:
Same here - probably better to have two actions - `enable_hook_for_repo`, `disable_hook_for_repo`.